nss-dynamic-elephants/Yak-Social-Network
# completed register form and added basic structure for validate module

## Description of Proposed Changes

Finished up input fields for the register form and added a module to validate a registration

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout register-validate-input
code .

```

1. run json server
1. npm start to view in browser
1. click on register button and see popup modal with registration form (first name, last name, email, location, password)
1. there is also "Validate" under the form which shows that our validate module is connected (no additional functionality for the validation yet). 

 


## Impacted Areas in Application

List general components of the application that this PR will affect:

* header
* registration


## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
